### ISSUE TITLE
Wrong type for build_absolute_uri function

### ISSUE THREAD

**Issue by mmmcorpsvit** [mirumee/saleor#2902]
The `# type` of `build_absolute_uri` function is outdated

```python
def build_absolute_uri(location):
    # type: (str, bool, saleor.site.models.SiteSettings) -> str  # - problem there
    host = Site.objects.get_current().domain
    protocol = 'https' if settings.ENABLE_SSL else 'http'
    current_uri = '%s://%s' % (protocol, host)
    location = urljoin(current_uri, location)
    return iri_to_uri(location)
```

**Comment by maarcingebala** [comment]
@mmmcorpsvit or @Pacu2 Could you edit the description to make it clear what needs to be fixed?

**Comment by Pacu2** [comment]
@maarcingebala done

**Comment by mmmcorpsvit** [comment]
comment there are doc-string, and have bad format

please see: 
https://docs.python.org/3/library/typing.html
or
https://www.python.org/dev/peps/pep-0257/

**Comment by mmmcorpsvit** [comment]
i think must be 
```python
 # type: (str) -> str
```

**Comment by maarcingebala** [comment]
Closed via #2929
